_posts/necessities/2018-02-05-chile-oil.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Chile Oil
+category: necessities
+---
+
+I always have chile oil in a jar in the pantry. My mother makes her own, and my recipe is an adaptation of hers. 
+
+- Start with whole, dried chilis. 
+
+- I live in California, and I prefer Mexican dried chilis which tend to be fresher than the ones found in Chinese grocery stores. Freshness is an interesting concept when it comes to something dried--I mean a chile that has a pliable quality to it still, and it isn't so brittle that it feels like dust waiting to happen in your hand. If you live somewhere with hotter summers than I do, you can grow your own peppers and dry them yourself to make chile oil. 
```

Comment:
'The dust waiting' could be made more direct: 'isn't so brittle that it crumbles to dust in your hand'.